browser/js/items/item_page.js

```diff
@@ -27,4 +28,5 @@ app.controller('ItemCtrl', function($scope, $state, $stateParams, SingleItemFact
 			$scope.product.sellerName = username;
```

Comment:
on line 25, can condense 25 and 26 to one line: $scope.showDescription = $scope.item.description != ""